src/api/app/controllers/webui/search_controller.rb

```diff
@@ -71,8 +71,8 @@ def set_parameters
     @search_where << 'description' if params[:description] == '1'
 
     @owner_limit = nil
-    @owner_limit = '1' if params[:limit].nil?
-    @owner_limit = params[:limit] unless params[:limit].nil?
+    @owner_limit = 1 if params[:limit].empty?
+    @owner_limit ||= Integer.try_convert(params[:limit]) || 1
```

Comment:
Isn't this just an on/off parameter? The value does not matter right?

Reply:
The value does not matter, yes. But if you do something like `/search/owner?limit=[1]` or `/search/owner?limit[]=1` it crashes.

Reply:
Sure I understand the bug but why not use `@owner_limit = params[:limit].present?`?

Reply:
Oh! I get it now. Right, we just store a boolean with that...

Reply:
oergs...